packages/features/bookings/lib/getBookingResponsesSchema.ts

```diff
@@ -80,6 +72,16 @@ function preprocess<T extends z.ZodType>({
           // If the field is not applicable in the current view, then we don't need to do any processing
           return;
         }
+        const fieldTypeSchema = fieldTypesSchemaMap[field.type as keyof typeof fieldTypesSchemaMap];
+        // TODO: Move all the schemas along with their respective types to fieldTypeSchema, that would make schemas shared across Routing Forms builder and Booking Question Formm builder
+        if (fieldTypeSchema) {
```

Comment:
Going to use this new approach for all existing fields, so that schema for each fieldType exists along with that fieldType.